src/Content/Widget/VCard.php

```diff
@@ -50,9 +51,15 @@ public static function getHTML(array $contact): string
 			Logger::warning('Incomplete contact', ['contact' => $contact ?? [], 'callstack' => System::callstack(20)]);
 		}
 
+		if (!Network::isValidHttpUrl($contact['url']) && Network::isValidHttpUrl($contact['alias'])) {
+			$url = $contact['alias'];
+		} else {
+			$url = $contact['url'];
```

Comment:
Please rename the variable to `$contact_url` for better visibility.
```suggestion
			$contact_url = $contact['alias'];
		} else {
			$contact_url = $contact['url'];
```